Makefile

```diff
@@ -1,10 +1,22 @@
-CFLAGS ?= -O3
+all:
 
-bf: bf.o
-	${CC} ${LDFLAGS} -o bf bf.o
+bf_base: bf_base.o
+	clang -o bf_base bf_base.o
 
-bf.o: bf.c
-	${CC} ${CFLAGS} -Wall -c bf.c
+bf_base.o: bf_base.c
+	clang -Wall -O3 -c bf_base.c
```

Comment:
Some targets have `-O3 ${CFLAGS}` and others only `-O3`. Is that intentional?

Reply:
Fixed in a1d505b.